src/settings.js

```diff
@@ -1516,6 +1516,14 @@ var USE_PTHREADS = false;
 // [compile+link] - affects user code at compile and system libraries at link.
 var WASM_WORKERS = 0;
 
+// If true, enables targeting Wasm Web Audio AudioWorklets.
```

Comment:
Maybe add a pointer here to read the docs under `site/../wasm_audio_worklets.rst`